include/benchmark/benchmark.h

```diff
@@ -667,14 +667,14 @@ class State {
   const int threads;
 
  private:
-  State(size_t max_iters, const std::vector<int64_t>& ranges, int thread_i,
+  State(IterationT max_iters, const std::vector<int64_t>& ranges, int thread_i,
```

Comment:
should we also define `Arg` or `Range` as a type instead of `int64_t` ?

Reply:
Hmm, possibly, but certainly not as `IterationCount`.